common/patches/rof2.cpp

```diff
@@ -36,6 +36,7 @@
 #include "../classes.h"
 #include "../races.h"
 #include "../raid.h"
+#include "../../zone/parcels.h"
```

Comment:
Probably want to avoid including a zone header in common, does it need a simple struct that can be moved to a common header?

Reply:
Removed parcels.h and place constants within eq_constants.